src/cascadia/WindowsTerminal/NonClientIslandWindow.h

```diff
@@ -95,4 +97,13 @@ class NonClientIslandWindow : public IslandWindow
     void _UpdateFrameMargins() const noexcept;
     void _UpdateMaximizedState();
     void _UpdateIslandPosition(const UINT windowWidth, const UINT windowHeight);
+
+    struct Revokers
+    {
+        winrt::event_token dragBar_SizeChanged;
+        winrt::event_token rootGrid_SizeChanged;
+        winrt::event_token titlebar_Loaded;
+        // LOAD BEARING!!
+        //If you add events here, make sure they're revoked in NonClientIslandWindow::Refrigerate

```

Comment:
see, autorevokers would make this much easier.. you could just set `_callbacks = {}`

Reply:
(I feel like that is a pattern we've used elsewhere to great effect)